client/src/partials/exchange/exchange.html

```diff
@@ -1,66 +1,130 @@
+                <td>{{ ExchangeCtrl.formatCurrency(rate.currency_id) }}</td>
+                <td>{{ rate.rate | currency:rate.currency_id }}</td>
+              </tr>
+            </tbody>
+          </table>
+        </div>  
+      </div>
+
+      <div class="col-xs-6" ng-switch="ExchangeCtrl.view">
+        <div class="panel panel-default" ng-switch-default>
+          <div class="panel-heading">
+            {{ "EXCHANGE.ENTERPRISE_CURRENCY" | translate }} {{ ExchangeCtrl.formatCurrency(ExchangeCtrl.enterprise.currency_id) }}
           </div>
+          <form name="RateForm" class="panel-body form-horizontal" ng-submit="ExchangeCtrl.submit(RateForm.$invalid)" novalidate>
+            <div class="form-group" >
+              <label>{{ "COLUMNS.DATE" | translate }}</label>
+              <input name="date" type="date" class="form-control" ng-class="{ 'ng-invalid-date' : RateForm.$submitted && RateForm.date.$invalid }" ng-model="ExchangeCtrl.form.date" required ng-disabled="!ExchangeCtrl.form.previous" ng-max="{{ ExchangeCtrl.tomorrow }}" max="{{ ExchangeCtrl.today | date:'yyyy-MM-dd' }}">
+            </div>
 
-          <div class="form-group">
-            <div class="checkbox">
-              <label>
-                <input name="previous" type="checkbox" ng-model="ExchangeCtrl.form.previous">
-                {{ "EXCHANGE.OLD_DATE" | translate }}
-              </label>
+            <div class="form-group">
+              <div class="checkbox">
+                <label>
+                  <input name="previous" id="previous" type="checkbox" ng-model="ExchangeCtrl.form.previous">
+                  {{ "EXCHANGE.OLD_DATE" | translate }}
+                </label>
+              </div>
             </div>
-          </div>
 
-          <div class="form-group" ng-repeat="(id, rate) in ExchangeCtrl.current">
-            <label class="control-label col-md-6">{{ ExchangeCtrl.formatCurrency(id) }}</label>
-            <div class="col-md-6">
-              {{ rate | currency:id }}
-              <a class="btn btn-warning text-right" ng-click="ExchangeCtrl.setExchangeRate(id)">{{ "FORM.UPDATE" | translate }}</a>
+            <div class="form-group" ng-repeat="(id, row) in ExchangeCtrl.current">
+              <label class="control-label col-md-6">{{ ExchangeCtrl.formatCurrency(id) }} </label>
+              <div class="col-md-6">
+                {{ row.rate | currency:id }}
+                <a class="btn btn-warning text-right" id="current-{{ $index + 1 }}" ng-click="ExchangeCtrl.setExchangeRate(id, row)">{{ "FORM.DEFINE" | translate }}</a>
+              </div>
+            </div>
+            <div ng-if="!ExchangeCtrl.current">
+              <div  class="form-group" ng-repeat="currency in ExchangeCtrl.outCurrencies">
+                <label class="control-label col-md-6">{{ ExchangeCtrl.formatCurrency(currency.id) }} </label>
+                <div class="col-md-6">
+                  {{ 0 | currency:currency.id }} 
```

Comment:
@jniles  I used zero if the exchange_rate table is empty and no exchange rate was reset